apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -428,7 +442,7 @@ const BookingPage = ({
     }
   };
 
-  const disableInput = !!rescheduleUid;
+  const disableInput = !!defaultUserValues.email && !!defaultUserValues.name;
```

Comment:
There was a case when no default user data was obtain neither query nor booking data, so input text field shouldn't be disabled in this case.